src/Instances/defaultProps.lua

```diff
@@ -99,5 +99,11 @@ return {
 		BackgroundColor3 = Color3.new(1, 1, 1),
 		BorderColor3 = Color3.new(0, 0, 0),
 		BorderSizePixel = 0
+	},
+	
+	CanvasGroup = {
+		BackgroundColor3 = Color3.new(1, 1, 1),
+		BorderColor3 = Color3.new(0, 0, 0),
+		BorderSizePixel = 0
 	}
```

Comment:
Maybe add a trailing comma to make diffs nicer in the future?

Reply:
I agree, a trailing comma here would be ideal.